boards/arm/tiva/tm4c1294-launchpad/Kconfig

```diff
@@ -72,4 +72,46 @@ config TM4C1294_LAUNCHPAD_TIMER_ALTCLK
 
 endif # TM4C1294_LAUNCHPAD_TIMER
 endif # TIVA_TIMER32_PERIODIC
+config TM4C1294_LAUNCHPAD_JUMPERS_CAN
+	bool "Use CAN jumpers configuration"
+	default n
+	select TIVA_UART2
+	---help---
+		If set, Nuttx will assume the jumpers JP4 and JP5 are set in CAN configuration.
```

Comment:
```suggestion
		If set, NuttX will assume the jumpers JP4 and JP5 are set in CAN configuration.

Reply:
Done in [997e60a](https://github.com/apache/nuttx/pull/8152/commits/997e60aaddec0694f4ca171eeeae0adc57b66ab3)